demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -883,6 +884,22 @@ def is_there_a_runnable(self) -> bool:
             return False
         return True
 
+    def is_valid_parameters_display_name(self) -> bool:
```

Comment:
docstring with good and bad examples will be nice.

Reply:
done